src/main/resources/locales/cy/ut-aac-weekly-hearing-list.json

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "Rhestr o Wrandawiadau Dyddiol Uwch Dribiwnlys (Siambr Apeliadau Gweinyddol)",
+    "heading": "Rhestr o Wrandawiadau Dyddiol Uwch Dribiwnlys (Siambr Apeliadau Gweinyddol)",
+    "message1": "Details",
+    "message2": "Lists are subject to change until 4:30pm. Any alterations after this time will be telephoned or emailed direct to the parties or their legal representatives.",
```

Comment:
Missing Welsh translation for these 2 lines. Same in data-management

Reply:
added missing welsh for both frontend and data management.